resources/views/admin.blade.php

```diff
@@ -225,9 +194,24 @@
                     });
                     table.draw(false);
                     $('.btn-close-dlt').click();
-                })
+                });
 
                 //hideLongText
+                function ellipsis(){
```

Comment:
remove addition functions like `ellipsis, truncateColumns`. using `hideLongText` method you can `substring` the values of  the data.

Reply:
Okay sure removed please review again @Raza9798 #60